BCStudentSoftwareDevTeam/lsf
# Fix supervisor portal errors

## Issue
Fixes issue #443 

There were some bugs and weird errors when using the supervisor portal like errors sorting, breaking presets, etc.

## Changes
- Removed evaluation functionality as that is no longer needed and throws errors.
- Stringified property attributes for Supervisor and Student sorting buttons to allow sorting to function properly.
- Added pre-student approval as a selected option to My Pending Forms.

## To Test
- Checkout `fix_supervisor_portal_errors`
- Verify that you can sort the results of form search by any of the columns.
- Make sure there are no traces of evaluations on the form search.
- Select My Pending Forms and verify that the "Pre-Student Approval" checkbox is automatically selected.
